.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md/
```

Comment:
- Good job for adding to `gitignore`, I think you added it after it was tracked by another commit 👌so make sure that this file is not present in your GitHub repository.
- Please untrack by git first then add it to the `.gitignore` file 😄
To untrack the file, you can use `git rm --cached test.md` in your terminal 👌😁

Reply:
Thank you for the feedback @binhussen
I ran git rm --cached test.md and it says the file is not being tracked anymore but it is still showing in the repository. What should I do to so that it doesn't show in the repository anymore?